tests/unit/s2n_tls13_handshake_test.c

```diff
@@ -470,43 +482,38 @@ int main(int argc, char **argv)
         EXPECT_EQUAL(s2n_conn_get_current_message_type(client_conn), ENCRYPTED_EXTENSIONS);
         EXPECT_SUCCESS(s2n_handshake_read_io(client_conn));
 
-        /* TODO add the rest of the handshakes tests
-         * (ServerCert, CertVerify, ServerFinished, ClientFinished)
-         * when respective TLS 1.3 components are done */
-
-        /*
-         * // Server sends ServerCert
-         * EXPECT_EQUAL(s2n_conn_get_current_message_type(server_conn), SERVER_CERT);
-         * EXPECT_SUCCESS(s2n_handshake_write_io(server_conn));
-         *
-         * // Server sends CertVerify
-         * EXPECT_EQUAL(s2n_conn_get_current_message_type(server_conn), SERVER_CERT_VERIFY);
-         * EXPECT_SUCCESS(s2n_handshake_write_io(server_conn));
-         *
-         * // Server sends ServerFinished
-         * EXPECT_EQUAL(s2n_conn_get_current_message_type(server_conn), SERVER_FINISHED);
-         * EXPECT_SUCCESS(s2n_handshake_write_io(server_conn));
-         *
-         * // Client reads ServerCert
-         * EXPECT_EQUAL(s2n_conn_get_current_message_type(client_conn), SERVER_CERT);
-         * EXPECT_SUCCESS(s2n_handshake_write_io(client_conn));
-         *
-         * // Client reads CertVerify
-         * EXPECT_EQUAL(s2n_conn_get_current_message_type(client_conn), SERVER_CERT_VERIFY);
-         * EXPECT_SUCCESS(s2n_handshake_write_io(client_conn));
-         *
-         * // Client reads ServerFinished
-         * EXPECT_EQUAL(s2n_conn_get_current_message_type(client_conn), SERVER_FINISHED);
-         * EXPECT_SUCCESS(s2n_handshake_write_io(client_conn));
-         *
-         * // Client sends ClientFinished
-         * EXPECT_EQUAL(s2n_conn_get_current_message_type(client_conn), CLIENT_FINISHED);
-         * EXPECT_SUCCESS(s2n_handshake_write_io(client_conn));
-         *
-         * EXPECT_EQUAL(s2n_conn_get_current_message_type(client_conn), APPLICATION_DATA);
-         * EXPECT_EQUAL(s2n_conn_get_current_message_type(server_conn), APPLICATION_DATA);
-         *
-         */
+        /* Client reads ServerCert */
+        /* Error is expected as the cert is untrusted */
```

Comment:
is this comment for line 397?

Reply:
Good catch! It was accidentally left in from a previous version and no longer applies.